libtransmission/announcer.c

```diff
@@ -153,6 +181,32 @@ typedef struct tr_announcer
 }
 tr_announcer;
 
+static struct tr_scrape_info* tr_announcerGetScrapeInfo(struct tr_announcer* announcer, const char* url)
+{
+    struct tr_scrape_info* info = NULL;
+
+    if (url && *url)
+    {
+        int pos;
+        bool found;
+        struct tr_scrape_info key;
+
+        key.url = (char*) url;
```

Comment:
This looks shorter and allows for `key` to be const:
```suggestion
        struct tr_scrape_info const key = { .url = (char*)url };
```

Reply:
Nice. Changed in 2817869f53578da6177e544059cc4dcb5bc5a731